spec/fixtures/templates/yml/invalid_myapp_vpc.yml

```diff
@@ -0,0 +1,6 @@
+---
+  Descr : : iption: "A test VPC template"
+this is not valid yml
+  Resources:
+    Vpc:
+      Type: "AWS::EC2::VPC"
```

Comment:
That's how it was used before. However, I (should have) removed it together
with the error handling code in Yaml.rb as I moved the error handling to
TemplateCompiler.

On Fri, 1 Apr 2016 at 15:17 Steve Hodgkiss <notifications@github.com> wrote:

> In spec/fixtures/templates/yml/invalid_myapp_vpc.yml
> <https://github.com/envato/stack_master/pull/95#discussion_r58160705>:
>
> > @@ -0,0 +1,6 @@
> > +---
> > +  Descr : : iption: "A test VPC template"
> > +this is not valid yml
> > +  Resources:
> > +    Vpc:
> > +      Type: "AWS::EC2::VPC"
>
> It could make a good test case :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/envato/stack_master/pull/95/files/0ccdb7c965eeb5af8a01c0097b237ea50aa9175f#r58160705>
>